apps/src/templates/sectionAssessments/StudentAssessmentOverviewTable.jsx

```diff
@@ -0,0 +1,174 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import commonMsg from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
+
+export const COLUMNS = {
+  QUESTION: 0,
+  STUDENT_ANSWER: 1,
+  CORRECT_ANSWER: 2,
+};
+
+const studentAnswerDataPropType = PropTypes.shape({
+  id:  PropTypes.string,
+  name: PropTypes.string,
+  studentAnswers: PropTypes.array,
+});
+
+const answerDataPropType = PropTypes.shape({
+  multipleChoiceOption: PropTypes.string,
+  percentAnswered: PropTypes.number,
+  isCorrectAnswer: PropTypes.bool,
+});
+
+const questionDataPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  question: PropTypes.string.isRequired,
+  answers: PropTypes.arrayOf(answerDataPropType),
+  notAnswered: PropTypes.number.isRequired,
+});
+
+class StudentAssessmentOverviewTable extends Component {
+  static propTypes= {
+    questionAnswerData: PropTypes.arrayOf(questionDataPropType),
+    studentAnswerData: PropTypes.arrayOf(studentAnswerDataPropType)
+  };
+
+  state = {
+    [COLUMNS.NAME]: {
+      direction: 'desc',
+      position: 0
+    }
+  };
+
+  getSortingColumns = () => {
+    return this.state.sortingColumns || {};
+  };
+
+  onSort = (selectedColumn) => {
+    this.setState({
+      sortingColumns: sort.byColumn({
+        sortingColumns: this.state.sortingColumns,
+        // Custom sortingOrder removes 'no-sort' from the cycle
+        sortingOrder: {
+          FIRST: 'asc',
+          asc: 'desc',
+          desc: 'asc'
+        },
+        selectedColumn
+      })
+    });
+  };
+
+  correctAnswerColumnFormatter = (answers, {rowData, columnIndex}) => {
+    const cell = rowData.answers;
+
+    let answerCell = '';
+      cell.forEach((answer) => {
```

Comment:
Nit: this block is indented unnecessarily. 

I also don't think this works if you have multiple correct answers. You could try using filter to get the correct answers, and then concat the strings together.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

Reply:
You could even pass an array of correct answers to your component for it to handle rendering.